README.md

```diff
@@ -89,7 +89,9 @@ To stop the server, use `httpd:stop()`.
 httpd = require('http.server').new(host, port[, { options } ])
 ```
 
-`host` and `port` must contain the interface and port to bind to.
+`host` and `port` must contain: 
+* For tcp socket the interface and port to bind to.
```

Comment:
interface → host (because, interface is wlan0 for example).

Reply:
Also add colon after 'For xxx socket', pls.

Reply:
Fixed